server/calendar/calendar.js

```diff
@@ -0,0 +1,61 @@
+var gcal = require('google-calendar');
+var calendarController = require('./calendar-controller');
```

Comment:
were you planning on using calendarController this to check whether a user is authenticated?